qmotif/src/org/qcmg/motif/Summariser.java

```diff
@@ -26,6 +26,7 @@
 import org.qcmg.common.string.StringUtils;
 import org.qcmg.common.util.LoadReferencedClasses;
 
+@Deprecated
```

Comment:
Just curious as to why this class (and its test ) is being deprecated?

Reply:
This class is not used anywhere, also can't find any related information in document.  It seems replaced by org.qcmg.motif.Motif,  class Motif is the mainClassName.

Reply:
> This class is not used anywhere, also can't find any related information in document. It seems replaced by org.qcmg.motif.Motif, class Motif is the mainClassName.

It's used to `Summarise` the `qmotif` xml files. The `Motif` class is performing a different function

Reply:
sure, I can remove de deprecated annotation if it is still used. However, could you create a related usage or document?